lib/rails/generators/intercom/config/intercom.rb.erb

```diff
@@ -21,8 +21,7 @@ IntercomRails.config do |config|
   <%- end -%>
 
   # == Intercom API Key
-  # This is required for some Intercom rake tasks like importing your users;
-  # you can generate one at https://app.intercom.io/apps/api_keys.
+  # You can generate one at https://app.intercom.io/apps/api_keys.
```

Comment:
Pretty sure, can't see it being used anywhere else https://github.com/intercom/intercom-rails/search?utf8=%E2%9C%93&q=api_key&type=Code